larpandoracontent/LArControlFlow/NeutrinoIdTool.h

```diff
@@ -24,13 +35,201 @@ class NeutrinoIdTool : public SliceIdBaseTool
      */
     NeutrinoIdTool();
 
-    void SelectOutputPfos(const SliceHypotheses &nuSliceHypotheses, const SliceHypotheses &crSliceHypotheses, pandora::PfoList &selectedPfos);
+    void SelectOutputPfos(const MasterAlgorithm *const pAlgorithm, const SliceHypotheses &nuSliceHypotheses, const SliceHypotheses &crSliceHypotheses, pandora::PfoList &selectedPfos);
 
 private:
+    class SliceFeatures;
```

Comment:
Probably don't need forward declaration here; can just move typedef under the full class definition.